src/redux/books/books.js

```diff
@@ -0,0 +1,38 @@
+const ADD_BOOK = 'redux/books/BOOK_ADDED';
+const REMOVE_BOOK = 'redux/books/BOOK_REMOVED';
+
+let currentId = 0;
+
+const reducer = (state = [], action) => {
+  switch (action.type) {
+    case ADD_BOOK:
+      return [
+        ...state,
+        {
+          id: action.id,
+          title: action.title,
+          author: action.author,
+          progress: 0,
+        },
+      ];
+    case REMOVE_BOOK:
+      return state.filter((book) => book.id !== action.id);
+    default:
+      return state;
+  }
+};
+
+export const addBook = (title, author) => ({
+  type: ADD_BOOK,
+  title,
+  author,
+  // eslint-disable-next-line
+  id: currentId++,
+});
+
+export const removeBook = (id) => ({
+  type: REMOVE_BOOK,
+  id,
```

Comment:
- [ ] [Optional] You created the reducers and action creators using the old way in ordinary `redux`, In `redux-toolkit` you suppose to use `createSlice` to write your reducer.
See this [link](https://redux-toolkit.js.org/api/createSlice) for more info.